### PR TITLE
Remove obs frame mechanism from the ioda reader

### PR DESCRIPTION
**Pull request by srherbener**
## Description

This PR contains test reference file changes in accordance with the removal of the obs frame mechanism in the ioda reader. See jcsda-internal/ioda/pull/913 for details.

### Issue(s) addressed

See jcsda-internal/ioda/pull/913

## Acceptance Criteria (Definition of Done)

All jedi-bundle ctests pass

## Dependencies

- [ ] merge with jcsda-internal/ioda/pull/913

## Impact

None

### CODE REVIEW

**Review by fmahebert — APPROVED** [review]

**Review by PatNichols — APPROVED** [review]
Looks good

**Comment by PatNichols** [comment]
@srherbener do I need to merge this since it's weird to merge your own PRs?

**Comment by srherbener** [comment]
> @srherbener do I need to merge this since it's weird to merge your own PRs?

No need, but thanks for checking. I think it's okay for me to merge this since it's been approved by others. I'm currently resolving some minor comments in the corresponding ioda PR so I plan to wait until that is finished and then merge the two PRs at the same time.

**Pull request merged**
